docs/user-guide/global-configurations/user-access.md

```diff
@@ -1,10 +1,50 @@
+
 # User Access
 
 {% embed url="https://www.youtube.com/watch?v=VTqBRIFbuSU" caption="" %}
 
-Authorization is used to determine what functions, data, or other parts of an application the user or the group should have access to.
-
-You can manage the User and Group access to Projects, Applications, Chart Groups, Environments, and Roles using the User Access feature.
+Like any entreprise product, Devtron supports fine grained access control to the resources based on
+1. Type of action allowed on the Devtron resources (Create Vs View)
+2. Sensitivity of the data (Editing image Vs Editing memory)
+
+Access can be added to the User either directly or via Groups.
+
+## Access Levels
+Devtron supports 5 levels of access
+1. **View**: User with `view` only access has least privilege. This user can only view combination of environments, applications and helm charts on which access has been granted to the user. This user cannot view sensitive data like secrets used in applications or charts.
+2. **Build and Deploy**: User with `build and deploy` permission can build and deploy the image of premitted applications to permitted environments.
+3. **Admin**: User with `admin` access can create, edit, delete and view permitted applications in permitted projects.
+4. **Manager**: User with `manager` access can do everything that an `admin` type user can do, in addition they can also give and revoke access of users for the applications and environments of which they are `manager`.
+5. **Super Admin**: User with `super admin` privilege has unrestricted access to all Devtron resources. Super admin can create, modify, delete and view any Devtron resource without any restriction; its like Superman without the weakness of Kryptonite. Super Admin can also add and delete user access across any Devtron resource, add delete git repository credentials, docker registry credentials, cluster and environment.
```

Comment:
jargons and manual !!!!!!!

Reply:
jargon nahi humour :) jargon is something totally different.. :) i think its ok over here